AirPushMac/Controllers/MainWindowController.swift

```diff
@@ -67,10 +71,19 @@ final class MainWindowController: NSWindowController {
             self?.viewModel.certificate = certificate
         }
     }
+    
+    func chooseKeyFile(_ sender: Any?) {
+        guard let window = self.window else { return }
+        chooseKeyFilePanel = ChooseKeyFilePanel()
+        chooseKeyFilePanel?.beginSheetModal(for: window) { [weak self] file in
```

Comment:
Why is the panel saved in the property? `ChooseCertificatePanel` worked without it